TwilioVerifySDKTests/TwilioVerify/Sources/API/FactorAPIClientTests.swift

```diff
@@ -89,7 +91,50 @@ class FactorAPIClientTests: XCTestCase {
     params.addAll(expectedParams)
     let createFactorPayload = CreateFactorPayload(friendlyName: Constants.friendlyName, type: Constants.factorType,
                                                   serviceSid: Constants.serviceSid, identity: Constants.identity,
-                                                  config: config, binding: binding, accessToken: Constants.accessToken)
+                                                  config: config, binding: binding, accessToken: Constants.accessToken,
+                                                  metadata: nil)
+    
+    factorAPIClient.create(withPayload: createFactorPayload, success: {_ in }, failure: {_ in })
+    
+    XCTAssertEqual(networkProvider.urlRequest!.url!.absoluteString, expectedURL,
```

Comment:
Can we use the  optional chain for this validations? I get that this is a test but I'm afraid that if this crashes due a null pointer exception we won't be able to find it easily on CircleCi, I have noticed that when something crashes during the CI execution, you can't see the specific cause

Reply:
Done for this line, but there are multiple lines using this